File: irshad dir/kamal/demo_java/java/L7/Prac2.java
//wajp to find occurances of element from the element provided by the user.
import java.io.*;
class Prac2
{
public static void main(String args[])
{
Console c=System.console();
int n=Integer.parseInt(c.readLine("Enter Number Of element In Array:"));
if(n<=0)
System.out.println("! Array Size cannot be 0 or negative. ");
else
{
int arr[]=new int[n];
for(int i=0;i<n;i++)
arr[i]=Integer.parseInt(c.readLine((i+1)+")"+"Enter Element:"));
int es=Integer.parseInt(c.readLine("Enter element to Search:"));
int oc=0;
for(int i=0;i<n;i++)
{
if (arr[i]==es){
oc++;
}
}
}
}
}

File: irshad dir/kamal/demo_python/python/L3/prac2.py
#wap to find the sum of digit
n=int(input("Enter number "))
if n<0:
	print("b +ve")
else:
	sum=0
	while n>0:
		d = n%10
		sum=sum+d
		n //= 10
	else:
		print(sum)